website/docs/components/form/masked-input/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,15 @@
+## Anatomy
+
+<!-- image then table -->
+![Anatomy of form MaskedInput](/assets/components/form/masked-input/masked-input-anatomy.png =836x*)
+
+<!-- this is just an example, refer to other components to see how to fill this table -->
+| Element          | Usage                                           |
+|------------------|-------------------------------------------------|
+| Label             | Required                                       |
+| Indicator         | Optional                                       |
+| Helper text       | Optional                                       |
+| Placeholder/Value | Optional                                       |
+| Toggle button     | Required                                       |
```

Comment:
Shouldn't this be optional? Are there use cases where the value should always be hidden?

Reply:
Confirmed this is in fact, a requirement and not optional. Will resolve this conversation.